packages/commonwealth/test/integration/api/thread.update.spec.ts

```diff
@@ -128,8 +128,8 @@ describe('Thread Patch Update', () => {
           .patch(`/api/threads/${thread.id}`)
           .set('Accept', 'application/json')
           .send({
-            author_chain: thread.chain,
-            chain: thread.chain,
+            author_chain: thread.community_id,
+            chain: thread.community_id,
```

Comment:
is there a plan to update api routes accepting `community` instead of `chain` as a param?

Reply:
Yes - this PR is only for model changes, not API changes (though a few might be included where necessary).